assets/src/scripts/monitoring-location/components/hydrograph/parameters.js

```diff
@@ -21,7 +21,7 @@ export const addSparkLine = function(svgSelection, {seriesLineSegments, scales})
         return;
     }
     let spark = line()
-        .x(function(d) {
+        .x(function(d){
```

Comment:
nit: we do leave in the space before the bracket. I will likely also add this to our lint rules (space-before-blocks)